skyrim/skyrim.py

```diff
@@ -20,26 +19,21 @@ def wrap_prediction(model_name, source):
         return GlobalEnsemblePrediction(source)
     else:
         raise ValueError("Invalid model name. Must be a string or a list of strings.")
-
-
+    
 class Skyrim:
-    def __init__(self, model_name: str):
-
-        self.model_name = model_name
-
-        if isinstance(model_name, list):
-            logger.info(f"Initializing ensemble model with {model_name}")
-            self.model = GlobalEnsemble(model_name)
-        elif isinstance(model_name, str):
-            if model_name in MODEL_FACTORY:
-                logger.debug(f"Initializing {model_name} model")
-                self.model = MODEL_FACTORY[model_name][0]()
-            else:
-                raise ValueError(f"Model name {model_name} is not supported.")
-        else:
-            raise ValueError(
-                "Invalid model name. Must be a string or a list of strings."
-            )
+    def __init__(self, *models: str, ic_provider: InitialConditionSource = InitialConditionSource.cds):
```

Comment:
It would enhance usability if the Skyrim class constructor allowed for the direct use of string literals when setting the `ic_provider.` Users should be able to initialize objects more intuitively, in my opinion.

Reply:
ok done makes sense.